### DISCUSSION TITLE
API pages inheritance arrow direction for modern template and markdown output format

### DISCUSSION THREAD

**Discussion by filzrev** [dotnet/docfx#9718] [Q&A]
It seems modern templates and markdown use `UML style` inheritance arrow direction.
https://dotnet.github.io/docfx/api/Docfx.Dotnet.DotnetApiOptions.html#inheritance

When using default template. It use `learn.microsoft.com` compatible inheritance arrows direction.
Is there any specific reason to change this style of `modern` template? (and `markdown` output format)

If these changes are intentional.
It might be better to support option to change arrow direction to `learn.microsoft.com` compatible format.

**Comment by yufeih** [comment]
Thanks you @just-ero for pointing out and @filzrev for creating a new thread.

I always thought the inheritance icon is left arrow but turns out to be right arrow on both MSLearn and the old template. Left arrow does feel more intuitive to me I wonder why it was otherwise (which perhaps @dend knows?)

**Reply by dend** [reply]
I think this is _mostly_ a left-over from the early implementation of inheritance logic for .NET documentation through DocFX 🙂 I agree that flipping the direction would be more intuitive.
